pdb/pom.xml

```diff
@@ -114,6 +121,8 @@
 			</plugin>
 		</plugins>
 	</build>
-
+	<properties>
+		<start-class>org.cbioportal.pdb_annotation.scripts.PdbAnnotationScriptsPipelineStarter</start-class>
```

Comment:
Yes, please let it as a command line now. I will make it running first.
Then I will change it back later.

On Fri, Jun 24, 2016 at 11:29 AM, onursumer <notifications@github.com>
wrote:

> In pdb/pom.xml
> <https://github.com/cBioPortal/pdb-annotation/pull/12#discussion_r68424185>
> :
>
> > @@ -114,6 +121,8 @@
> >  			</plugin>
> >  		</plugins>
> >  	</build>
> > -
> > +	<properties>
> > +		<start-class>org.cbioportal.pdb_annotation.scripts.PdbAnnotationScriptsPipelineStarter</start-class>
>
> Instead of tweaking the current pdb module by this way, we should add
> another module. This turns the current web application into a command line
> tool where we can only execute a single script.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cBioPortal/pdb-annotation/pull/12/files/58ed4383e3651e3b3e4c3a053323fd95da79bad4#r68424185>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADL7zMd3TYiBaxFPg5bfMh4WesN97kVbks5qPAYGgaJpZM4I5V0q>
> .
>